pom.xml

```diff
@@ -156,6 +156,11 @@
     </dependency>
     <dependency>
       <groupId>org.apache.commons</groupId>
+      <artifactId>commons-collections4</artifactId>
```

Comment:
Got it. No problem. Apparently the Guava folks don't like this, but I do: https://github.com/google/guava/issues/796 :)